CMakeLists.txt

```diff
@@ -42,42 +43,70 @@ message(STATUS "")
 # External
 # ============================================================================ #
 
+include(ExternalProject)
+find_program(AUTORECONF_EXE NAMES autoreconf DOC "autoreconf is required to build CUDD")
+find_program(MAKE_EXE NAMES gmake make DOC "make is required to build CUDD")
+
 # ---------------------------------------------------------------------------- #
 # BDD Packages
 # Adiar Package (Steffan Sølvsten - Aarhus University, Denmark)
 if (BDD_BENCHMARK_STATS)
   set(ADIAR_STATS ON)
 endif(BDD_BENCHMARK_STATS)
 
-add_subdirectory (external/adiar adiar)
+add_subdirectory (external/adiar adiar EXCLUDE_FROM_ALL)
 
 # BuDDy Package (Jørn Lind-Nielsen - Copenhagen University, Denmark)
 set(BUDDY_EXAMPLES OFF)
 if (BDD_BENCHMARK_STATS)
   set(BUDDY_STATS ON)
 endif(BDD_BENCHMARK_STATS)
 
-add_subdirectory (external/BuDDy bdd)
+add_subdirectory (external/BuDDy bdd EXCLUDE_FROM_ALL)
 
 # Cal Package (Jagesh Sanghavi, Rajeev Ranjan et al. - University of California, United States of America)
-add_subdirectory (external/cal cal)
+add_subdirectory (external/cal cal EXCLUDE_FROM_ALL)
+
+# CUDD Package (Fabio Somenzi - University of Colorado Boulder, United States of America)
+set(CUDD_SRC_DIR ${PROJECT_SOURCE_DIR}/external/cudd)
+set(CUDD_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/cudd-build)
+set(CUDD_INSTALL_DIR ${CMAKE_CURRENT_BINARY_DIR}/cudd-install)
+if (BDD_BENCHMARK_STATS)
+  string(APPEND CUDD_CPPFLAGS -DDD_STATS)
+endif(BDD_BENCHMARK_STATS)
 
-# Cudd Package (Fabio Somenzi - University of Colorado Boulder, United States of America)
```

Comment:
This line propably should be preserved

Reply:
The diff is misleading, it is still there at line 70. I only changed “Cudd” to “CUDD.”

Reply:
Yes, I noticed the same after making the comment. I've forwarded the CUDD repo and just testing it works. Then it will be in.